### DISCUSSION TITLE
Topology plugin issue with Netbox 3.0.6

### DISCUSSION THREAD

**Discussion by phsantos71** [netbox-community/netbox#7524] [Q&A]
Hi,

Im new on Netbox and after install a fresh copy o Netbox and Topology plugin, Im getting the following error to run this plugin:

_Server Error
There was a problem with your request. Please contact an administrator.

The complete exception is provided below:

<class 'django.template.exceptions.TemplateDoesNotExist'>

base.html

Python version: 3.9.2
NetBox version: 3.0.6_

My Netbox version is 3.0.6

Could someone help me to solve this issue?

Thanks

Paulo Santos

![Screenshot 2021-10-11 200325](https://user-images.githubusercontent.com/26817639/136865248-db2469a2-10d4-427d-9672-e865dfdb99df.png)

**Comment by kkthxbye-code** [comment]
Netbox Topology does not support netbox v3 yet.

Please see:
https://github.com/mattieserver/netbox-topology-views/issues/72

https://github.com/mattieserver/netbox-topology-views/pull/79

**Comment by kurt00ar** [comment]
I have the same issue... =(
